lib/dry/inflector/inflections/dictionary/uncountable.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+module Dry
+  class Inflector
+    class Inflections
+      module Dictionary
+        module Uncountable
+          def self.list
```

Comment:
WDYT about simply setting `Uncountable` as a constant instead of having a method?

Reply:
That could work as well 😄. I mention the option for the file based on the idea that would be easier to maintain in the long run

Reply:
Thanks for reply!  I will make it.